configuration/components/token-refresher.libsonnet

```diff
@@ -0,0 +1,50 @@
+local defaults = {
+  namespace: error 'must provide namespace',
+  serviceMonitor: true,
+  secretName: 'token-refresher-oidc',
```

Comment:
nit (feel free to ignore): suggest to move `serviceMonitor` and `secretName` to the bottom since they are already populated with default values.